main.go

```diff
@@ -779,6 +779,20 @@ func initialiseSystem(arguments map[string]interface{}) error {
 		afterConfSetup(&config.Global)
 	}
 
+	if os.Getenv("TYK_LOGLEVEL") == "" && arguments["--debug"] == false && config.Global.LogLevel != "" {
+		switch strings.ToLower(config.Global.LogLevel) {
+		case "error":
+			log.Level = logrus.ErrorLevel
+		case "warn":
+			log.Level = logrus.WarnLevel
+		case "debug":
+			log.Level = logrus.DebugLevel
+		}
+		log.WithFields(logrus.Fields{
+			"prefix": "main",
+		}).Debugf("Using log level defined in config. Enabling %s-level output", strings.ToLower(config.Global.LogLevel))
```

Comment:
I'd say this is a bit silly, given how it will only do something if the debug level is debug. Remove? Or just move it to the `"debug"` case.

Reply:
On second thought, I'd say just remove. If you set the debug log level, you'll see tons of debug logs anyway.